packages/sampler/stories/default/ViewManager.js

```diff
@@ -0,0 +1,134 @@
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, number, select} from '@enact/storybook-utils/addons/knobs';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import Group from '@enact/ui/Group';
+import Item from '@enact/ui/Item';
+import Layout, {Cell} from '@enact/ui/Layout';
+import Scroller from '@enact/ui/Scroller';
+import {arrange, SlideArranger, SlideBottomArranger, SlideLeftArranger, SlideRightArranger, SlideTopArranger, ViewManager} from '@enact/ui/ViewManager';
+import {useCallback, useState} from 'react';
+
+import css from './ViewManager.module.less';
+
+const ViewManagerConfig = mergeComponentMetadata('ViewManager', ViewManager);
```

Comment:
Let's add `VIewManagerBase` for default props.

Reply:
I've added it.